compaction/compaction_manager.cc

```diff
@@ -753,6 +753,8 @@ future<> compaction_manager::rewrite_sstables(column_family* cf, sstables::compa
                     });
                 }
                 _stats.completed_tasks++;
+                // Schedule regular compaction on rewrite completion, to allow it to catch up with new sstables produced.
+                postpone_compaction_for_column_family(&cf);
```

Comment:
@raphaelsc please review

I think we need (in addition and independently) tie-in between compaction manager and sstables manager. If compaction manager senses a large backlog in the number of sstables, it should postpone open-for-write (other than those needed for compaction). This stops creating new sstables until compaction catches up.

It will be an interesting exercise to implement it, since sstables manager and compaction manager don't know about each other (and shouldn't).